testframework/src/main/java/net/neoforged/testframework/conf/FrameworkConfiguration.java

```diff
@@ -16,13 +16,19 @@
 import net.minecraft.resources.ResourceLocation;
 import net.neoforged.testframework.impl.MutableTestFramework;
 import net.neoforged.testframework.impl.TestFrameworkImpl;
+import net.neoforged.testframework.summary.SummaryFormatter;
 import org.jetbrains.annotations.Nullable;
 
 @ParametersAreNonnullByDefault
 @MethodsReturnNonnullByDefault
 public record FrameworkConfiguration(
-        ResourceLocation id, Collection<Feature> enabledFeatures, int commandRequiredPermission,
-        List<String> enabledTests, @Nullable Supplier<ClientConfiguration> clientConfiguration) {
+        ResourceLocation id,
+        Collection<Feature> enabledFeatures,
+        int commandRequiredPermission,
+        List<String> enabledTests,
+        @Nullable Supplier<ClientConfiguration> clientConfiguration,
+        boolean useDefaultFormatters,
```

Comment:
No point in adding this boolean, just make the formatters list default to the defaults.

Reply:
I didn't want to reference the impl package from the outside

Reply:
That's fine to do, the annotation is for consumers not under our control.